public/script.js

```diff
@@ -5,8 +5,8 @@ const fqdn = window.location.host;
 
 const DEFAULT_EXPIRY = 60 * 60; // one hour
 const PASSPHRASE_CHARSET =
-  "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz@#$%^&?";
```

Comment:
Yeah, I'm not sure exactly what's going on but most likely you aren't rebasing off the most up-to-date master... you may want to recreate your master

Reply:
I would recommend recreating master and then cherry picking your changes onto a newly created branch